docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -227,151 +222,165 @@ Follow these steps:
 2. Build the PyTorch Docker image.
 
    ```bash
-   cd.circleci/docker
-   ./build.sh pytorch-linux-bionic-rocm<version>-py3.7
-   # eg. ./build.sh pytorch-linux-bionic-rocm3.10-py3.7
+   cd .ci/docker
+   ./build.sh pytorch-linux-<os-version>-rocm<rocm-version>-py<python-version> -t rocm/pytorch:build_from_dockerfile
    ```
 
-   This should be complete with a message "Successfully build `<image_id>`."
+   Where:
+   * `<os-version>`: `ubuntu20.04` (or `focal`), `ubuntu22.04` (or `jammy`), `centos7.5`, or `centos9`
+   * `<rocm-version>`: `5.4`, `5.5`, or `5.6`
+   * `<python-version>`: `3.8`-`3.11`
+
+   To verify that your image was successfully created, run:
+
+   `docker image ls rocm/pytorch:build_from_dockerfile`
+
+   If successful, the output looks like this:
 
-3. Start a Docker container using the image:
+   ```bash
+   REPOSITORY    TAG                       IMAGE ID         CREATED           SIZE
+   rocm/pytorch  build_from_dockerfile     17071499be47     2 minutes ago     32.8GB
+   ```
+
+3. Start a Docker container using the image.
 
    ```bash
-   docker run -it --cap-add=SYS_PTRACE --security-opt
-   seccomp=unconfined --device=/dev/kfd --device=/dev/dri --group-add
-   video --ipc=host --shm-size 8G <image_id>
+   docker run -it --cap-add=SYS_PTRACE --security-opt --user root \
+   seccomp=unconfined --device=/dev/kfd --device=/dev/dri \
+   --group-add video --ipc=host --shm-size 8G \
+   -v ~/pytorch:/pytorch rocm/pytorch:build_from_dockerfile
    ```
 
-   You can also pass -v argument to mount any data directories from the host
-   onto the container.
+   You can also pass the `-v` argument to mount any data directories from the host onto the container.
 
-4. Clone the PyTorch repository.
+4. Go to the PyTorch directory.
 
    ```bash
-   cd ~
-   git clone https://github.com/pytorch/pytorch.git
    cd pytorch
-   git submodule update --init --recursive
    ```
 
-5. Build PyTorch for ROCm.
+5. Set ROCm architecture. The Docker image tag is `rocm/pytorch:latest-base`.
```

Comment:
@LisaDelaney `Set ROCm architecture` is a mandatory step here
Full text (feel free to rephrase):
5. Set ROCm architecture

To determine your AMD architecture, run:

```bash
rocminfo | grep gfx
```

The result will be like this and your architecture is "gfx1030" for example:

```text
Name:                    gfx1030
 Name:                    amdgcn-amd-amdhsa--gfx1030
```

Set the PYTORCH_ROCM_ARCH environment variable to specify which architectures to build PyTorch for

```bash
export PYTORCH_ROCM_ARCH=<uarch>
```

where `<uarch>` is the architecture reported by the `rocminfo` command above.